contrib/system_containers/centos/config.json.template

```diff
@@ -413,6 +413,17 @@
             "source": "/dev",
             "type": "bind"
         },
+        {
+            "destination": "/host",
+            "options": [
+                "rbind",
+                "rshared",
+                "rw",
+                "mode=777"
```

Comment:
This raises a red flag?  Does this effect anything?

Reply:
I added this in preparation of userNS, so that all the files under '/host' are visible in the system container as they are on the host without any change done by the mount itself.